README.md

```diff
@@ -49,12 +49,45 @@ functions:
 
 Make sure to configure your GitHub App registration's webhook URL to `<your lambda's URL>/api/github/webhooks`.
 
+## Test deployment
+
+For testing your Probot deployment without end to end (GitHub) integration, you can use this shell snippet:
+
+```shell
+$ LAMBDA_URL=https://x.execute-api.y.amazonaws.com/stage-or-basePath/api/github/webhooks
+$ SECRET=the_webhook_secret
+$ TMP_DATA_FILE=/tmp/smoke.data
+
+$ echo -n "{\"action\":\"test\"}" > $TMP_DATA_FILE
+$ SIGN=$(openssl dgst -sha1 -hmac $SECRET $TMP_DATA_FILE | cut -d" " -f2)
+$ curl --request POST --header "X-Hub-Signature: sha1=$SIGN" --header "X-Github-Event: test" --header "X-GitHub-Delivery: fake" --data-binary "@$TMP_DATA_FILE" $LAMBDA_URL
+{"ok":true}        <-- Concent for Probot v10: {"message":"Received test.test"}
+```
+
 ## Examples
 
 - [example-aws-lambda-serverless](https://github.com/probot/example-aws-lambda-serverless/#readme) - Official example application that is continuously deployed to AWS Lambda
 
 Add yours!
 
+## Common errors
+
+| Http Code | Message | Description
+| --- | --- | --- |
+| 403 | Missing Authentication Token | Bad endpoint (this one is not binded on Lambda) |
+| 500 | Internal server error | Incorrect headers value (`X-GitHub-Delivery`, `X-GitHub-Event`, `X-Hub-Signature`) or Probot error |
```

Comment:
can you run `npm run lint:fix`? That will format the markdown tables

Reply:
Done